content/pages/education/other-educational-assistance-programs/call-to-service.md

```diff
@@ -44,6 +44,8 @@ You may chose from the following:
 
 ### How do I get these benefits?
 
+Apply online with VA Form 22-1990N. [Find out how to apply](/education/apply/). 
```

Comment:
@goldenmeanie Forgot to say, please link the whole line as Rachael did above in her last couple of commits: [Apply online with VA Form 22-1990N]. Thanks!